client/src/admin/NotificationsPage.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NotificationsPage = () => (
+  <div className="container--content">
```

Comment:
Why temp content? Shouldn't we design what the table is going to look like? For all our other pages we put together the UI before we had a server. We should do the same here. I suggest creating a fake server response and using that to populate real views.

If you'd like that to be another PR that's fine. This one looks good as is.